common/src/com/elikill58/negativity/common/commands/NegativityCommand.java

```diff
@@ -327,18 +328,36 @@ public boolean onCommand(CommandSender sender, String[] arg, String prefix) {
 	}
 	
 	private void sendHelp(CommandSender sender) {
-		Messages.sendMessageList(sender, "negativity.verif.help");
+		if (Perm.hasPerm(sender, Perm.VERIF))
+			Messages.sendMessageList(sender, "negativity.verif.help");
+		if (Perm.hasPerm(sender, Perm.CHECK))
+			Messages.sendMessageList(sender, "negativity.help");
+		if (sender instanceof Player)
+			Messages.sendMessageList(sender, "negativity.alert.help");
+		if (!(sender instanceof Player) || Perm.hasPerm(sender, Perm.RELOAD))
```

Comment:
People which are not player have all permissions (such as console sender) ?

Reply:
This condition is the inversion of https://github.com/Elikill58/Negativity/blob/44e6fea69aa47fed34dba196de34ea0c9e6bcd4e/common/src/com/elikill58/negativity/common/commands/NegativityCommand.java#L119

Reply:
Yes but in this case, the `!(sender instanceof Player)` isn't necessary for me.